### DISCUSSION TITLE
aquamarine and WLR_DRM_NO_ATOMIC

### DISCUSSION THREAD

**Discussion by AndreasYNY** [hyprwm/Hyprland#7186] [Q&A] [answered]
Hi all,

As `WLR_DRM_NO_ATOMIC` abuser to play osu (at least up until the 6.9, after that tearing feels off), will aquamarine/hyprland support or do something similar? or should i move back to X? (since kwin tearing doesn't work on my hardware i doubt other wayland environments would work either).

Thank you.

<details close>
<summary>(Not sure if i should leave my hardware here, but here it is )</summary>

```
Host: Aspire A315-41 (V1.18)
Kernel: Linux 6.10.2-zen1-1-zen
CPU: AMD Ryzen 5 2500U with Radeon Vega Mobile Gfx (8) @ 2.00
GPU: AMD Radeon Vega 8 Graphics [Integrated]
```

```
Hyprland, built from branch main at commit 2b520571e897be2a0e88c8692da607b062000038  (keybinds: improve fullscreenstate toggling (7174)).
Date: Mon Aug 5 02:40:34 2024
Tag: v0.41.2-165-g2b520571, commits: 5051

flags: (if any)
```

</details>

**Comment by vaxerski** [comment, marked as answer]
`AQ_NO_ATOMIC`, but _not_ recommended as it is not very well supported.
